src/Microsoft.Extensions.FileSystemGlobbing/Abstractions/DirectoryInfoBase.cs

```diff
@@ -2,16 +2,32 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
-using System.IO;
 
 namespace Microsoft.Extensions.FileSystemGlobbing.Abstractions
 {
+    /// <summary>
+    ///     Represents a directory
```

Comment:
Leaves the comments extra hard to maintain for anyone else who may touch this class but (like many) doesn't use ReSharper. We use VS defaults and avoid random alignments for a reason.